lib/resolvers/user_mutations/blockForPlugin.js

```diff
@@ -1,14 +1,23 @@
 const { NotFoundError } = require('errors');
 const User = require('../../models/User');
 const userExists = require('../../helper_functions/userExists');
-const requestContext = require('talawa-request-context');
+const requestContext = require("../../helper_lib/talawa-request-context/talawa-request-context");
 const superAdminCheck = require('../../resolvers/functions/superAdminCheck');
+const i18n = require('i18n');
+const express = require('express');
 
-module.exports = async (parent, args, context) => {
+const app = express();
+
+
+const blockforPlugin = async (parent, args, context) => {
+  app.use(i18n.init);
   let userFound = await userExists(args.userId);
   if (!userFound) {
     throw new NotFoundError(
-      requestContext.translate('user.notFound'),
+      // requestContext.translate('user.notFound'),
```

Comment:
likewise, similar to the above, remove the commented code

Reply:
ok